lib/studies/reply/inbox.dart

```diff
@@ -27,27 +27,34 @@ class InboxPage extends StatelessWidget {
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
               Expanded(
-                child: ListView(
-                  padding: EdgeInsetsDirectional.only(
-                    start: startPadding,
-                    end: endPadding,
-                    top: isDesktop ? 28 : 0,
-                  ),
-                  children: [
-                    for (int index = 0;
-                        index < model.emails[destination].length;
-                        index++) ...[
-                      MailPreviewCard(
-                        id: index,
-                        email: model.emails[destination][index],
-                        onDelete: () => model.deleteEmail(destination, index),
-                        onStar: () => model.starEmail(destination, index),
+                child: model.emails[model.currentlySelectedInbox].isEmpty
+                    ? Center(
+                        child: Text(
+                          '${model.currentlySelectedInbox} inbox is empty.',
```

Comment:
What are the other inboxes called? Does this sentence still work with those names too?

Reply:
Ahh thanks for catching this. It would work for every inbox except "Inbox", then we get `Inbox inbox is empty`.